src/rcc.rs

```diff
@@ -143,6 +151,32 @@ impl CFGR {
         self
     }
 
+    /// Gives write access to the registers in the backup domain
+    pub fn enable_backup_domain(&mut self, pwr: &mut PWR) -> BackupDomain {
+        // NOTE: Access to apb1enr inside Rcc struct is means we have exclusive access
```

Comment:
Sentence doesn't parse for me

Reply:
:+1: That "is" was definitively not supposed to be there. I updated the wording for both comments.

I also changed the name of `enable_backup_domain` to `access_backup_domain` which is more descriptive of what's actually happening